docs/configuration.md

```diff
@@ -75,9 +75,6 @@ The configuration uses the following structure:
     label: function () {
       return `${this.config.partnerId} | ${this.config.uiConfId ? `${this.config.uiConfId} | ` : ''}${this.config.entryId} | '${this.config.entryName}'`
     },
-    value: function () {
```

Comment:
Better to just point to default tracking file instead of holding double reference